### ISSUE TITLE
Unexpected behaviour when logging out 'undefined'

### ISSUE THREAD

**Issue by mattgrande** [ramda/repl#49]
*Steps to reproduce*

1. Go the the repl
2. Enter this: `const z = [null, undefined]`
3. `console.log(z)`

*Expected result*

You should see `[null, undefined]` in the right-hand side

*Actual result*

You see `[null, null]`

*Browser:* Chrome 59.0.3071.115
*OS:* OSX 10.12.5 (Sierra)

Kinda strange; I'm mostly just curious if others have seen this issue.

**Comment by buzzdecafe** [comment]
yeah that is strange. It appears to be just in the way `console.log` works. If you just do:

```js
const z = [null, undefined]
z;
```

the right side of the repl responds with `[null, undefined]`. Since it evaluates correctly, this doesn't strike me as a crisis, just another hairy js edge.
